### PR TITLE
fix for CICD to enable 7.1.0-snapshot release

### PR DESCRIPTION
**Pull request by mdebarros**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -630,10 +630,10 @@ workflows:
           context: org-global
           requires:
             - setup
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
+#            - test-unit
```
Comment (lewisdaly) [inline .circleci/config.yml]: Is this a temporary measure?

File: .circleci/config.yml [hunk 2]
```diff
@@ -651,89 +651,14 @@ workflows:
...
-                - /.*/
       - build:
           context: org-global
           requires:
             - setup
             - test-unit
             - test-coverage
-            - test-integration
-            - test-functional
+#            - test-integration
```
Comment (lewisdaly) [inline .circleci/config.yml]: Same as above

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 [![Docker pulls](https://img.shields.io/docker/pulls/mojaloop/bulk-api-adapter.svg?style=flat)](https://hub.docker.com/r/mojaloop/bulk-api-adapter)
 [![CircleCI](https://circleci.com/gh/mojaloop/bulk-api-adapter.svg?style=svg)](https://circleci.com/gh/mojaloop/bulk-api-adapter)
 
-Bulk Transfers API and notifications
+Bulk Transfers API and notifications.
```
Comment (lewisdaly) [inline README.md]: Good.

File: package.json [hunk 4]
```diff
@@ -39,16 +40,20 @@
     "pre-commit": "1.2.2",
     "standard": "12.0.1",
     "swagmock": "1.0.0",
-    "tap-xunit": "2.4.1",
+    "tap-xunit": "^2.4.1",
```
Comment (lewisdaly) [inline package.json]: Please remove 🥕

File: test/unit/data/transfers/{id}/error.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 'use strict'
-var Mockgen = require('../../mockgen.js')
+var Mockgen = require('../../mockgen')
```
Comment (lewisdaly) [inline test/unit/data/transfers/{id}/error.js]: Should this be `let` or `const`?

File: test/unit/data/mockgen.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 var Swagmock = require('swagmock')
```
Comment (lewisdaly) [inline test/unit/data/mockgen.js]: Should these be `let` or `const`?

File: test/unit/transfers/{id}/error.test.js [hunk 7]
```diff
@@ -4,7 +4,8 @@ const Test = require('tape')
 const Hapi = require('hapi')
 const HapiOpenAPI = require('hapi-openapi')
 const Path = require('path')
-const Mockgen = require('../../../data/mockgen.js')
+const Mockgen = require('../../data/mockgen')
+// const imp = require('../../../../src/interface/swagger.yaml')
```
Comment (lewisdaly) [inline test/unit/transfers/{id}/error.test.js]: Remove unneeded comment.

File: .circleci/config.yml (hunk 1 above)
Comment (mdebarros) [reply .circleci/config.yml]: Correct. I have only disabled tests on snapshots.

PRs and tagged releases will still have to pass tests.

File: package.json (hunk 4 above)
Comment (mdebarros) [reply package.json]: Fixed

File: test/unit/data/transfers/{id}/error.js (hunk 5 above)
Comment (mdebarros) [reply test/unit/data/transfers/{id}/error.js]: I have changed this, but note that these are auto-generated by swag plugin

File: test/unit/data/mockgen.js (hunk 6 above)
Comment (mdebarros) [reply test/unit/data/mockgen.js]: Same, auto-generated by swaggerize. I have fixed it though.

File: test/unit/data/transfers/{id}/error.js (hunk 5 above)
Comment (lewisdaly) [reply test/unit/data/transfers/{id}/error.js]: Ah, I see. That makes sense.

**Comment by lewisdaly** [comment]
Code looks good, but the unit tests are failing.

**Comment by mdebarros** [comment]
> Code looks good, but the unit tests are failing.

That is correct. There are none except the ones auto-generated from the swaggerize plugin. For now, we will release this as a snapshot until we have all the unit/QA tests in place.

**Pull request merged**